wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/header.php

```diff
@@ -20,5 +20,11 @@
 
 get_template_part( 'header', 'wporg' );
 ?>
+
+<script>
+	// used in `message.js`
```

Comment:
nothing like passing php vars to javascript. brings back lots of memories for me.